### DISCUSSION TITLE
Running Jamulus server on AWS EC2

### DISCUSSION THREAD

**Discussion by mes1mak** [jamulussoftware/jamulus#2771] [Technical Support & Help]
Is there a procedure and recommendation on how to run Jamulus server on AWS EC2?

**Comment by henkdegroot** [comment]
You can find a few guides and even youtube videos for this. Here a link to one of these: https://jal25.medium.com/creating-and-running-a-jamulus-server-in-amazon-ec2-a-practical-guide-c7c1305e0f93

I strongly recommend to use the Jamulus installation process as documented here (using the .deb image): https://jamulus.io/wiki/Running-a-Server

The headless .deb is ideal for this.
